### ISSUE TITLE
Deployment issues

### ISSUE THREAD

**Issue by bkak** [mspnp/microservices-reference-implementation#217]
I am following this deployment guide: https://github.com/mspnp/microservices-reference-implementation/blob/main/deployment.md and upon running this command on Ubuntu 20.04 I get the following error:

Command: 
```
az deployment group create -f ./workload/workload-stamp.json -g $RESOURCE_GROUP -p droneSchedulerPrincipalId=$DRONESCHEDULER_ID_PRINCIPAL_ID -p workflowPrincipalId=$WORKFLOW_ID_PRINCIPAL_ID -p deliveryPrincipalId=$DELIVERY_ID_PRINCIPAL_ID -p acrResourceGroupName=$RESOURCE_GROUP_ACR
```

Error:
```
{"error":{"code":"InvalidTemplate","message":"Deployment template language expression evaluation failed: 'Unable to parse language expression 'concat('ai-', uniqueString(resourceGroup().id)': expected token 'RightParenthesis' and actual 'EndOfData'.'. Please see https://aka.ms/arm-template-expressions for usage details.","additionalInfo":[{"type":"TemplateViolation","info":{"lineNumber":0,"linePosition":0,"path":""}}]}}
```
Could you please guide me on how to fix this?

**Comment by ckittel** [comment]
@magrande missing a `)` at the end of that line? Was that one of the recent changes?

**Comment by ckittel** [comment]
Thanks for letting us know, this has been resolved in #218
